backend/module/eCampCore/src/Acl/AclFactory.php

```diff
@@ -155,6 +158,7 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
                 CampCollaboration::class,
                 Period::class,
                 Day::class,
+                DayResponsible::class,
```

Comment:
FETCH_ALL is allowed for all, FETCH only if in camp?

Reply:
ja.
 - FetchAll darfst du, sobald du angemeldet bist (nicht Gast)
 - Fetch darfst du nur, wenn du die Entität sehen kannst